src/wp-includes/rest-api.php

```diff
@@ -1393,6 +1393,11 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
 					return new WP_Error( 'rest_invalid_param', sprintf( __( '%s is not a valid IP address.' ), $param ) );
 				}
 				break;
+			case 'uuid':
+				if ( ! wp_is_uuid( $value ) ) {
+					return new WP_Error( 'rest_invalid_uuid', __( 'Invalid UUID' ) );
```

Comment:
Let's include the parameter name in the error message like we do for other validations.